Makefile

```diff
@@ -84,11 +84,11 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 		rbac:roleName=manager-role output:rbac:artifacts:config=config/components/rbac\
 		crd:generateEmbeddedObjectMeta=true output:crd:artifacts:config=config/components/crd/bases\
 		webhook output:webhook:artifacts:config=config/components/webhook\
-		paths="./..."
+		paths="{./api/..., ./pkg/...}"
```

Comment:
Note: I had to change some Makefile rules to target specific paths rather than everything (`./...`) since it was picking up some javascript dependencies in the site/ directory, now that it is populated.